src/axes.jl

```diff
@@ -732,7 +774,10 @@ function axis_drawing_info(sp, letter)
                 end
             end
 
-            if !(ax[:minorticks] in (:none, nothing, false)) || ax[:minorgrid]
+            if (
+                (!(ax[:minorticks] in (:none, nothing, false)) || ax[:minorgrid]) &&
+                !isnothing(minor_ticks)
```

Comment:
Added a guard against minor_ticks === nothing.

Reply:
```suggestion
                minor_ticks !== nothing
```

Reply:
Is there a reason for not using the builtin `isnothing` ?

Reply:
the compiler loves `===` checks

Reply:
Updated.